spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -2920,7 +2936,8 @@ object QueryPlanSerde extends Logging with CometExprShim {
       val canSort = sortOrder.head.dataType match {
         case _: BooleanType => true
         case _: ByteType | _: ShortType | _: IntegerType | _: LongType | _: FloatType |
-            _: DoubleType | _: TimestampType | _: TimestampType | _: DecimalType | _: DateType =>
+            _: DoubleType | _: TimestampType | _: TimestampNTZType | _: DecimalType |
```

Comment:
This is unrelated to the goal of the PR but I noticed we had TimestampType twice and no TimestampNTZType